lib/wallaby/browser.ex

```diff
@@ -410,6 +410,34 @@ defmodule Wallaby.Browser do
     parent
   end
 
+  @doc """
+  Executes javascript asynchronously, taking as arguments the script to execute,
```

Comment:
I looked into it and I don't think it executes the script asynchronously, when executing the script the WebDriver client will block until the callback is called. Maybe we could reword this?